exporters/user_events/src/recordable.cc

```diff
@@ -0,0 +1,115 @@
+/// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "opentelemetry/exporters/user_events/logs/recordable.h"
+#include "opentelemetry/sdk/common/global_log_handler.h"
+
+#include <chrono>
+#include <map>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace user_events
+{
+namespace logs
+{
+
+namespace api_logs = opentelemetry::logs;
+
+Recordable::Recordable() noexcept
+{
+  event_builder_.Reset("OpenTelemetry-Logs");
```

Comment:
Isn't the event's name supposed to be used here?

Reply:
Currently it is hard-coded here. Where do you suggest to move it to?

Reply:
I think this is supposed to be the OpenTelemetry event name. I believe you get that from SetEventId.